postgres/src/lib.rs

```diff
@@ -82,6 +89,7 @@ pub struct Manager {
 }
 
 impl Manager {
+    #[cfg(feature = "runtime")]
```

Comment:
Do you really need to feature gate all of this? As the feature gated functions are generic over `T` it shouldn't cause any harm, does it? Is the core of the issue the `tokio::spawn` which isn't supported by the WASM target?

Reply:
So this one (and its friends) need to be gated because they all initialize the Postgres connection _through_ the `tokio_postgres::Config`. That's what's not supported when you don't have the `runtime` feature on `tokio_postgres` (like on WASM), since it requires `tokio_postgres` to establish the TCP connection on your behalf. So in a world without `runtime`, we can only keep around the APIs where you bring your own connection.